xmodule/seq_block.py

```diff
@@ -557,10 +557,8 @@ def _get_render_metadata(self, context, children, prereq_met, prereq_meta_info,
                 'This section is a prerequisite. You must complete this section in order to unlock additional content.'
             )
 
-        blocks = self._render_student_view_for_blocks(context, children, fragment, view) if prereq_met else []
-
         params = {
-            'items': blocks,
+            'items': self._render_student_view_for_blocks(context, children, fragment, view),
```

Comment:
Are we sure this doesn't break the prerequisites feature altogether? Why is this the proper fix to the issue?